subsys/logging/log_backend_swo.c

```diff
@@ -91,7 +91,8 @@ static void log_backend_swo_init(void)
 	/* Enable unprivileged access to ITM stimulus ports */
 	ITM->TPR  = 0x0;
 	/* Configure Debug Watchpoint and Trace */
-	DWT->CTRL = 0x400003FE;
+	DWT->CTRL &= 0x000003FF;
```

Comment:
Could you make further improvement and use CMSIS mask and bit fieds here?

Reply:
Yes, i just pushed the changes.